tests/robustness/validate/validate.go

```diff
@@ -58,6 +58,12 @@ type Config struct {
 	ExpectRevisionUnique bool
 }
 
+func prepareAndCategorizeOperations(reports []report.ClientReport) ([]porcupine.Operation, []porcupine.Operation) {
+	patchedReport := patchFailedRequestWithInfiniteReturnTime(reports)
+	return relevantOperations(patchedReport), filterSerializableOperations(patchedReport)
```

Comment:
Please don't patch report, rewrite function to one loop to return operations.

Reply:
Returning operation now!